build.gradle

```diff
@@ -11,8 +11,6 @@ repositories {
 
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
-}
-
-sourceSets {
-    main.kotlin.srcDirs += 'src/main/kotlin'
```

Comment:
Why did you remove the sourceSets block?

Reply:
It shouldn't be needed AFAIK as you are using the default convention (https://kotlinlang.org/docs/reference/using-gradle.html#targeting-the-jvm)
I can keep it in though as one could argue it is more declarative.

Reply:
I'm fine with removing it then, thanks!